transfermethodui/src/main/java/com/hyperwallet/android/ui/transfermethod/view/AddTransferMethodFragment.java

```diff
@@ -68,11 +70,14 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Locale;
+import java.util.Map;
 import java.util.TreeMap;
 
 public class AddTransferMethodFragment extends Fragment implements WidgetEventListener, AddTransferMethodContract.View {
 
-    public static final String TAG = AddTransferMethodFragment.class.getName();
+    protected static final String TAG = "transfer-method:add:collect-transfer-method-information";
+    protected static final String TAG_ADDED = "transfer-method:add:transfer-method-created";
```

Comment:
can we rename this to `TAG_TRANSRFER_METHOD_ADDED` and make it private, since we only need 1 protected tag for each fragment